core/templates/core/base_international.html

```diff
@@ -0,0 +1,101 @@
+{% load static from staticfiles %}
+
+<!DOCTYPE html>
+<!--[if lt IE 9]><html class="lte-ie8" lang="en"><![endif]-->
+<!--[if gt IE 8]><!--><html lang="en"><!--<![endif]-->
+ <head>
+    <meta id="utmCookieDomain" value="{{ analytics.UTM_COOKIE_DOMAIN }}" />
+    {% include 'core/google_tag_manager_head.html' %}
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta property="og:image" content="{% static 'images/icons/opengraph-image.png' %}">
+    <title>{% block head_title %}International - GREAT.gov.uk{% endblock %}</title>
+
+    <link rel="shortcut icon" href="{% static 'images/icons/favicon.ico' %}" type="image/x-icon" />
+    <link rel="mask-icon" href="{% static 'images/icons/gov.uk_logotype_crown.svg' %}" color="#0b0c0c">
+    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="{% static 'images/icons/apple-touch-icon-152x152.png' %}">
+    <link rel="apple-touch-icon-precomposed" sizes="120x120" href="{% static 'images/icons/apple-touch-icon-120x120.png' %}">
+    <link rel="apple-touch-icon-precomposed" sizes="76x76" href="{% static 'images/icons/apple-touch-icon-76x76.png' %}">
+    <link rel="apple-touch-icon-precomposed" href="{% static 'images/icons/apple-touch-icon-60x60.png' %}">
+
+    <link rel="stylesheet" type="text/css" href="/static/styles/international.css">
+    {% block css %}{% endblock %}
+
+    <script src="{% static 'Dit-Pattern-Styling/public/js/third-party.js' %}"></script>
+    <script src="{% static 'Dit-Pattern-Styling/public/js/script.js' %}"></script>
+    <script src="{% static 'js/international.js' %}"></script>
+    {% block scripts %}{% endblock %}
+
+</head>
+<body id="top">
+    {% include 'core/google_tag_manager_body.html' %}
+    <header id="header">
+        <a id="skip-link" href="#content" class="verbose">Skip to main content</a>
+        <section role="dialog" data-component="language-selector-dialog" data-lang="en" data-label="International" data-name="English" class="language-selector-dialog">
+            <div data-node="languages" class="languages">
+                <p id="great-languages-selector">Choose your language</p>
+                <ul aria-labelledby="great-languages-selector" class="navigation">
+                <li><a href="?lang=en">English</a></li>
```

Comment:
these should all lead to`/international/??lang=`

Reply:
The currently specified href value means it would just add ?lang=xx param to current URL, but have updated code to reflect your preferred "/international/?lang=" specific location.